core/templates/pages/contributor-dashboard-page/modal-templates/translation-modal.component.spec.ts

```diff
@@ -208,26 +208,44 @@ describe('Translation Modal Component', () => {
   });
 
   describe('when clicking on the translatable content', () => {
-    let target: HTMLElement;
+    const nonParagraphTarget: HTMLElement = document.createElement('div');
+    const mathTarget: HTMLElement = document.createElement(
+      'oppia-noninteractive-math');
+    let paragraphTarget: HTMLElement;
     let broadcastSpy: jasmine.Spy<(target: HTMLElement) => void>;
     let propagationSpy: jasmine.Spy<() => void>;
     beforeEach(fakeAsync(() => {
+      paragraphTarget = document.createElement('p');
```

Comment:
I think I have already initialized `paragraphTarget` in beforeEach? Is this what you mean @nithusha21?